paddle/fluid/operators/ngraph/ngraph_engine.h

```diff
@@ -12,12 +12,18 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
+#ifndef PADDLE_FLUID_OPERATORS_NGRAPH_NGRAPH_ENGINE_H_
```

Comment:
use `pragma once` instead.

Reply:
will do. Maybe later PR (not sure if it will dismiss your approval if I do it in this PR)

Reply:
You can address this in #15752 .